src/cmd/manage_nsfs.js

```diff
@@ -148,8 +150,10 @@ async function fetch_bucket_data(argv, config_root, from_file) {
             versioning: 'DISABLED',
             path: bucket_path,
             should_create_underlying_storage: true,
+            creation_date: creation_date,
         };
     }
+    update_bucket_creation_date(data);
```

Comment:
Why do you set the `creation_date` in the `data` and then update it (`update_bucket_creation_date`)?
You can update it and then set it in the data, the function would return the date instead of changing the property in place, for example:
```suggestion
   creation_date: get_updated_bucket_creation_date(creation_date),
```

Reply:
we can't do that because we need to do the same logic for --from-file flow too.